src/app/website/posts/receipt-chain.markdown

```diff
@@ -0,0 +1,71 @@
+---
+title: How to make a scalable blockchain (1), techniques of SNARK 
+How do you prove you sent someone a transaction when there's no blockchain?
+date: 2018-12-12
+---
+
+As we wrap up 2018 at O(1) Labs, we thought it would be fun to look back on
+the year's developments, highlighting some interesting aspects TODO
+
+Let's imagine Alice and Bob have gone to the grocery store. Alice gets a jar
+of peanut butter and a bunch of bananas, the total ringing up as 8 Coda.
+Bob gets some cheddar cheese and a loaf of bread, which also ends up costing
+8 Coda. They both go to pay the grocery store at their separate cashiers at
+about the same time. But now there seems to be an issue, the grocer's balance
+has gone up by only 8 Coda: one of Alice's or Bob's transactions failed to go
+through.
+
+Alice needs to get to basketball practice and Bob is meeting his friend Carol to
+eat the bread, so each is anxious to get out of there. So, each wants to convince
+the grocer that their transaction was the one that went through.
+
+If Alice and Bob were using a heavy blockchain (TODO) the answer would be clear: just have the grocer look
+through the transaction history on the blockchain to see which was sent. But Alice
+and Bob are using Coda, where there is no history stored on the blockchain. How can
+they convince the grocer that their transaction was sent?
+
+Let's discuss some potential solutions, and then how Coda actually solves this problem.
+
+# Potential solution 1: One grocer account per cashier
```

Comment:
it would be nice to include diagrams for each of the potential solutions. I can help out and draw up some stuff in tikz if you'd like, just let me know.